CONTRIBUTING.md

```diff
@@ -98,7 +95,7 @@ emmake make install
 
```

Comment:
I figured out this flaw in my PR too. Now that we use the patched emsdk we need to use its sysroot !

So we can do 
```
export BUILD_PREFIX=$MAMBA_ROOT_PREFIX/envs/xeus-cpp-wasm-build
export SYSROOT_PATH=$BUILD_PREFIX/opt/emsdk/upstream/emscripten/cache/sysroot
```
The sysroot will be present in the opt (optional folder that takes care of such third party things we need)